index.html

```diff
@@ -5,25 +5,35 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" type="text/css" href="style.css">
+        <link rel="stylesheet" type="text/css" href ="desktop.css">
         <title>Portfolio Setup Mobile First</title>
     </head>
     <body>
         <!--Header section-->
         <header class="header">
-            <a id="logo" href="#">Allan K. </a>
-            <nav>
+            <nav class="mobile-menu">
+                <a class="logo" href="#">Allan K. </a>
                 <a href="#"><img class="menuimg" src="assets/header/menu.png" alt="menu"/></a>
             </nav>
+            <nav class="menu-desktop">
+                <a class="logo" href="#">Allan K. </a>
+                <ul class="nav-links">
+                    <li class="nav-link"><a href="#portfolio">Portfolio</a></li>
+                    <li class="nav-link"><a href="#about">About</a></li>
+                    <li class="nav-link"><a href="#feedback">Contact</a></li>
+                    <li><a href="#"><img src="assets/header/Icon - Mail.svg" alt="mail"></a></li>
+                </ul>
+            </nav>           
```

Comment:
kindly ensure the logo is in the same vertical alignment with the headline section as it is in the design given so you'd have a good design 😄 . Also, I'd suggest you move the navigation items away from the border line a bit by adding some margin 👍 

Figma design :
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/20537712/47257a18-5cc3-4531-bba8-000fad3a5908)

Your design :
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/20537712/f786d344-a81c-4078-84d7-953166c330f8)